packages/123done/static/js/123done.js

```diff
@@ -17,7 +17,8 @@ $(document).ready(function () {
   const paymentURL = {
     local: 'http://localhost:3030/subscriptions/products/',
     dev: 'https://latest.dev.lcip.org/subscriptions/products/',
-    stage: 'https://accounts.stage.mozaws.net/subscriptions/products/',
+    stage:
+      'https://payments-stage.fxa.nonprod.cloudops.mozgcp.net/subscriptions/products/',
```

Comment:
Should this be changed from a content-server URL to a payments server URL directly?

Currently all RPs link to the content-server first, which then redirects to the appropriate payments server URL.

Reply:
That's a good question. I am not sure - should the URLs within `paymentURL` be payments server URLs (`local` and `prod` would need to be updated)? We have `contentURL` below with the content-server URLs.

cc: @biancadanforth

Reply:
Yes I believe Reino is correct here; RPs should always be linking to the content server redirect URL, so the content server can check authentication state, metrics opt-out preferences and add flow metrics as needed.